frontend/src/app/header/header.component.html

```diff
@@ -42,12 +42,14 @@
       id="menu-bar"
       fxLayout="row"
     >
-      <app-logo variant="light"></app-logo>
+      <app-logo variant="dark"></app-logo>
+      <span fxFlex></span>
       <div
         id="primary-nav"
         *ngIf="!mobileQuery.matches"
       >
-        <a
+        <ul>
+          <li><a
           mat-button
           [routerLink]="[ '/enroll' ]"
```

Comment:
Once we created the About page, this needs to route there

Reply:
Seems like we need to think through this a little more with the group.  I'm going to accept this as is, going to the "enroll" page.  But that page seems to have all the content for the about page.  I'm going to make a note to rename this page.

Reply:
I renamed the "enroll" page "about" and fixed all the links so that things make sense here. We already have a "Register" page and the term "enroll" is used extensively in the context of being already registered and adding participants to your account.  So I think this change makes everything a little clearer.  But I've been wrong.  From time to time.